opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -74,20 +75,37 @@ class BrineCo2Pvt
 public:
     using H2O = SimpleHuDuanH2O<Scalar>;
     using Brine = ::Opm::BrineDynamic<Scalar, H2O>;
-    using CO2 = ::Opm::CO2<Scalar>;
+    using CO2 = ::Opm::CO2<Scalar, Params>;
 
     //! The binary coefficients for brine and CO2 used by this fluid system
     using BinaryCoeffBrineCO2 = BinaryCoeff::Brine_CO2<Scalar, H2O, CO2>;
 
     explicit BrineCo2Pvt() = default;
 
-    explicit BrineCo2Pvt(const std::vector<Scalar>& salinity,
+    explicit BrineCo2Pvt(const ContainerT& salinity,
                          int activityModel = 3,
                          int thermalMixingModelSalt = 1,
                          int thermalMixingModelLiquid = 2,
                          Scalar T_ref = 288.71, //(273.15 + 15.56)
                          Scalar P_ref = 101325);
 
+    explicit BrineCo2Pvt(ContainerT brineReferenceDensity,
+                        ContainerT co2ReferenceDensity,
+                        ContainerT salinity,
+                        int activityModel,
+                        Co2StoreConfig::SaltMixingType thermalMixingModelSalt,
+                        Co2StoreConfig::LiquidMixingType thermalMixingModelLiquid,
+                        Params params) :
```

Comment:
Hard to read. Move colon to next line, and reindent that section, preferably with commas aligned with the colon (I have come to like that as it makes deleting or adding arguments a 1-line diff).